### DISCUSSION TITLE
Has  anyone used a RS-485 Surge Supressor?

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#855] [Q&A]
I'm running a Pentair ET currently.  It has blown several 485 chips during thunderstorms, usually during  very brief power fluctuations.  I recently saw this  beast & was  wondering if anyone has  used something like it & what your experience has been.  

https://www.serialcomm.com/serial_rs232_converters/opto_isolated_and_surge_suppressors/rs485_surge_suppressor/rs485_surge_suppressor.product_general_info.aspx

I am planning to build a  Nixie this fall so I'm hoping it's  more robust than the ET.  Still wondering if the surge suppressor might be worth including in the Nixie build.

Thanks for your thoughts!

**Comment by gw8674** [comment]
What I'm using...
https://www.amazon.com/Tupavco-Ethernet-Protector-Gigabit-1000Mbs/dp/B00805VUD8/ref=sr_1_2?crid=3F1RDBLORS98B&keywords=Tupavco+TP302&qid=1693180643&sprefix=tupavco+tp302+%2Caps%2C160&sr=8-2

One between my Arris Cable Modem and my router/firewall, and the rest up with the attic patch panels protecting my various runs out to the pool equipment pad (network and RS-485 runs).

**Comment by FlaMike** [comment]
Thanks for your response.  I am running a pair of optical media converters (link below) between my cable modem & router to protect things from surges to the ISP connection.  I also run the Ethernet from the pool pad through the surge suppressor of a UPS.  I'm not sure if that approach is as good as the Ethernet protector you have for pool pad isolation--the most likely remaining source of surges from the outside world.  I'm looking to protect the RS-485 equipment on the pad from surges.  I ordered the beast linked above & hoping it will protect the not-so-great ET circuitry until the Nixie build is installed.  And I plan to keep it in place to provide additional protection for the Nixie.

https://www.amazon.com/Gigabit-Ethernet-Converter-1000Base-Tx-Transceiver/dp/B06XZ7HXMR/ref=sr_1_3?crid=G0UMTIL6Y9LW&keywords=10gtek%2Bgigabit%2Bethernet%2Bmedia%2Bconverter&qid=1696277918&sprefix=10gtek%2Bgigabit%2Bmedia%2Bconverter%2Caps%2C172&sr=8-3&th=1

**Comment by FlaMike** [comment]
For what it's worth, this morning I installed the RS-485 surge suppressor linked in my first message in this thread to my ET4 that's controlled through njs-PC.  In my installation, I have a 485 to Ethernet converter in the ET to a docker box that's running njs-PC with a USB converter.  The surge suppressor was simple enough to install.  Hopefully, the infamous ET RS-485 problems are history.  The ET will be eventually be history as well, once my Nixie is fired up--sometime this autumn.
